packages/core/src/build/plugin.ts

```diff
@@ -18,6 +18,10 @@ export default schema;
 
 const apiModule = () => `import { createPublicClient } from "viem";
 
+if (globalThis.PONDER_INDEXING_BUILD === undefined || globalThis.PONDER_DATABASE === undefined) {
+  throw new Error('Invalid dependency graph. Please ensure that nothing is imported from "src/api/index.ts"')
```

Comment:
Invalid dependency graph. Config, schema, and indexing function files cannot import objects from the API function file "src/api/index.ts".